### DISCUSSION TITLE
Is it possible to access a context variable in a js expression to evaluate the conditions of a switch?

### DISCUSSION THREAD

**Discussion by diego-034** [elsa-workflows/elsa-core#1118] [Q&A]
I want to be able to choose the path of the switch in content previously saved in variables, is it possible?

![imagen](https://user-images.githubusercontent.com/54601468/121586877-cb502900-c9f9-11eb-8a1f-1725bc1437e9.png)

## Error

```sh
Failed to evaluate expression workflowContext.getVariable("answerTree") == 0 using syntax JavaScript <s:Elsa.Expressions.ExpressionEvaluator>
```

Could you give me an example of a use from the designer either with js or liquid?

Very Thanks!!! nice job!!

**Comment by sfmskywalker** [comment]
If `answerTree` is a variable you defined earlier (using either the `setVariable` JS function or `SetVariable` activity), then you should be able to access the variable directly, so try this:

```js
answerTree == 1
```

Notice no quotes used - workflow variables are real JS variables.
